doc/source/whatsnew/v0.23.0.txt

```diff
@@ -386,9 +386,7 @@ Conversion
 - Bug in :class:`WeekOfMonth` and :class:`LastWeekOfMonth` where default keyword arguments for constructor raised ``ValueError`` (:issue:`19142`)
 - Bug in localization of a naive, datetime string in a ``Series`` constructor with a ``datetime64[ns, tz]`` dtype (:issue:`174151`)
 - :func:`Timestamp.replace` will now handle Daylight Savings transitions gracefully (:issue:`18319`)
-
-
```

Comment:
pls leave white space this is on purpose to avoid conflicts

Reply:
Gotcha, will fix.